testing/src/main/java/io/stargate/it/http/RestApiv2Test.java

```diff
@@ -691,6 +691,38 @@ public void createTableWithNullOptions() throws IOException {
     assertThat(successResponse.getSuccess()).isTrue();
   }
 
+  @Test
+  public void createTableMissingClustering() throws IOException {
+    createKeyspace(keyspaceName);
+    TableAdd tableAdd = new TableAdd();
+    tableAdd.setName(tableName);
+
+    List<ColumnDefinition> columnDefinitions = new ArrayList<>();
+
+    columnDefinitions.add(new ColumnDefinition("pk1", "int"));
+    columnDefinitions.add(new ColumnDefinition("ck1", "int"));
+
+    tableAdd.setColumnDefinitions(columnDefinitions);
+
+    PrimaryKey primaryKey = new PrimaryKey();
+    primaryKey.setPartitionKey(Collections.singletonList("pk1"));
+    primaryKey.setClusteringKey(Collections.singletonList("ck1"));
+    tableAdd.setPrimaryKey(primaryKey);
+
+    tableAdd.setTableOptions(new TableOptions(0, null));
+
+    String body =
+        RestUtils.post(
+            authToken,
+            String.format("%s:8082/v2/schemas/keyspaces/%s/tables", host, keyspaceName),
+            objectMapper.writeValueAsString(tableAdd),
+            HttpStatus.SC_CREATED);
+
+    SuccessResponse successResponse =
+        objectMapper.readValue(body, new TypeReference<SuccessResponse>() {});
+    assertThat(successResponse.getSuccess()).isTrue();
```

Comment:
I don't see how that test verifies if the clustering was created with `ASC` or `DESC`. Would it be possible to assert it here?

Reply:
It's not in the response but I can turn around and get the table just to be safe. The point of this test was that before the change this request would have failed.